bson-adt-core/src/test/scala/adt/bson/scalacheck/BsonValueGenerators.scala

```diff
@@ -167,11 +113,12 @@ trait BsonValueGenerators extends RegexGenerators {
     }
   }
 
-  implicit val shrinkBsonObject: Shrink[BsonObject] = Shrink { bson =>
-    shrink[Map[String, BsonValue]](bson.value) map { fields =>
-      BsonObject(fields)
-    }
-  }
+  // TODO: Fix
+//  implicit val shrinkBsonObject: Shrink[BsonObject] = Shrink { bson =>
+//    shrink[Map[String, BsonValue]](bson.value) map { fields =>
+//      BsonObject(fields)
+//    }
+//  }
```

Comment:
This was sometimes causing an infinite loop in the tests. I'll see if this is still the case.